server/account.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"chai/database/sqlc"
+)
+
+type accountRes struct {
+	Message string       `json:"message"`
+	Data    sqlc.Account `json:"accounts"`
+}
+
+func (a *App) AccountHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodPost:
+		a.InsertAccountHandler(w, r)
+	case http.MethodGet:
+		a.SelectAccountHandler(w, r)
+	default:
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+	}
+}
+
+func (a *App) InsertAccountHandler(w http.ResponseWriter, r *http.Request) {
+	var params sqlc.InsertAccountParams
+	params.Owner = r.URL.Query().Get("name")
+	params.Currency = r.URL.Query().Get("currency")
+	params.Balance = 0
+	newAccount, err := a.Queries.InsertAccount(context.Background(), params)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	_ = json.NewEncoder(w).Encode(
+		accountRes{
+			Message: "Account created successfully",
+			Data:    newAccount,
+		})
+
+}
+
+func (a *App) SelectAccountHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Same down here, make it private.

```suggestion
func (a *App) selectAccountHandler(w http.ResponseWriter, r *http.Request) {
```